### ISSUE TITLE
Recipe system.

### ISSUE THREAD

**Issue by Scremonetu** [SkQuery/SkQuery#5]
- Skript version: The latest from Spigotmc.org (3.6.3)
- Server software: Spigot
- Server version: PaperSpigot 1.13.2

```
[00:28:45] [Server thread/WARN]: java.lang.Throwable: Warning: A plugin is creating a recipe using a Deprecated method. This will cause you to receive warnings stating 'Tried to load unrecognized recipe: bukkit:<ID>'. Please ask the author to give their recipe a static key using NamespacedKey.
[00:28:45] [Server thread/WARN]:     at org.bukkit.inventory.ShapedRecipe.<init>(ShapedRecipe.java:28)
[00:28:45] [Server thread/WARN]:     at com.w00tmast3r.skquery.elements.effects.EffNewRecipe.execute(EffNewRecipe.java:55)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.lang.Effect.run(Effect.java:52)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.effects.Delay$1.run(Delay.java:109)
[00:28:45] [Server thread/WARN]:     at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:82)
[00:28:45] [Server thread/WARN]:     at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:449)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1003)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:938)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836)
[00:28:45] [Server thread/WARN]:     at java.lang.Thread.run(Thread.java:748)
[00:28:45] [Server thread/WARN]: java.lang.Throwable: Warning: A plugin is creating a recipe using a Deprecated method. This will cause you to receive warnings stating 'Tried to load unrecognized recipe: bukkit:<ID>'. Please ask the author to give their recipe a static key using NamespacedKey.
[00:28:45] [Server thread/WARN]:     at org.bukkit.inventory.ShapedRecipe.<init>(ShapedRecipe.java:28)
[00:28:45] [Server thread/WARN]:     at com.w00tmast3r.skquery.elements.effects.EffNewRecipe.execute(EffNewRecipe.java:55)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.lang.Effect.run(Effect.java:52)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[00:28:45] [Server thread/WARN]:     at ch.njol.skript.effects.Delay$1.run(Delay.java:109)
[00:28:45] [Server thread/WARN]:     at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:82)
[00:28:45] [Server thread/WARN]:     at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:449)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1003)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:938)
[00:28:45] [Server thread/WARN]:     at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836)
[00:28:45] [Server thread/WARN]:     at java.lang.Thread.run(Thread.java:748)

```

**Comment by TheLimeGlass** [comment]
Recipe system needs to be recoded.

**Comment by Scremonetu** [comment]
Any update on this?

**Comment by Viljami0** [comment]
Is it still in the works?
Or is there another addon that does not spam the chat?

**Comment by TheLimeGlass** [comment]
Fixed in 4.1.2, but I will not be remaking the recipe system at this time. You can use another addon that does handle recipes properly.
